packages/angular/src/app/shared/types/contact.ts

```diff
@@ -50,7 +50,7 @@ export const newContact: Contact = {
   email: '',
   image: '',
   address: '',
-  zipCode: '',
+  zipCode: null,
```

Comment:
It is weird: zipCode is null in data and it is number in types at the same time and it works without errors

Reply:
i've remake type